Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDllHelpers/DebuggeFlowController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using DbgEngManaged;
+using System;
+
+namespace CsDebugScript.Engine.Debuggers.DbgEngDllHelpers
+{
+    /// <summary>
+    /// Controler for Debugee actions during live debugging.
+    /// </summary>
+    class DebuggeeFlowController
```

Comment:
true

---
In reply to: [70291777](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70291777) [](ancestors = 70291777,69980965)